lib/portage/output.py

```diff
@@ -522,16 +522,19 @@ def get_term_size(fd=None):
         fd = sys.stdout
     if not hasattr(fd, "isatty") or not fd.isatty():
         return (0, 0)
-    try:
-        import curses
 
+    # Do not use curses by default because it returns stale values after terminal resize.
+    if cached:
```

Comment:
This leaves behind dead code, but maybe it will be fixed in curses at some point in the future. Meanwhile, it serves as documentation of why not to use curses here.

Reply:
Interesting. I found https://stackoverflow.com/a/53019662/194894, which suggests that we should use

```
curses.initscr().getmaxy()
```

instead of `curses.tigetnum()`.

Reply:
Hmm, I tested this, and it seems `curses.initscr()` has some unwanted side-effects.

Reply:
I've removed to cached parameter and just comments about not using curses here.